modules/API/API.js

```diff
@@ -89,6 +89,11 @@ function initCommands() {
 
             APP.store.dispatch(muteAllParticipants(localIds));
         },
+        'enable-lobby': () => {
+            const { conference } = APP.store.getState()['features/base/conference'];
+
+            conference.enableLobby();
```

Comment:
Please don't do this, use the `toggleLobbyMode` action and dispatch it.

Reply:
Done. That action is now dispatched when the command is used.